pkg/auth/prometheus/metrics.go

```diff
@@ -0,0 +1,36 @@
+package prometheus
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	AuthSubsystem = "auth_subsystem"
```

Comment:
Private unless there is another reason.  

Use "openshift_auth" to better describe (subsystem is redundant in public metrics